backend/tests/restrictionTests.test.js

```diff
@@ -0,0 +1,80 @@
+//import axios from 'axios';
+var axios = require('axios')
+
+jest.mock('axios');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
```

Comment:
Codacy has a fix for the issue: [Replace `'http://20.53.224.7:8081'` with `"http://20.53.224.7:8081"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
const UserDBURL = "http://20.53.224.7:8081";
```